scripts/lib/exec.ts

```diff
@@ -27,50 +27,29 @@ export async function exec(
 ): Promise<string> {
 	nicelog(`> $ ${command} ${args.join(' ')} (in ${pwd}))`)
 	return new Promise((resolve, reject) => {
-		const data: string[] = []
-
-		const childProcess = execFile(
-			command,
-			args.filter((arg): arg is string => !!arg),
+		const childProcess = execute(
+			`${command} ${args.filter((arg): arg is string => !!arg).join(' ')}`,
```

Comment:
`execFile` was used here because it sanitizes the arguments passed to it. Can we restore sanitization here?